### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *quiet-gelato-475ecd* failed.

### ISSUE THREAD

**Issue by hockhocking** [HustleHacktory/example-nextjs-2f718#23]
### <span aria-hidden="true">❌</span> Deploy Preview for *quiet-gelato-475ecd* failed.

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 46a4c0a69693a803191410b4893b0642596799b4 |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/projects/quiet-gelato-475ecd/deploys/6a8dd75913e8da0008b1535b |

_Originally posted by @hockhocking in https://github.com/HustleHacktory/example-nextjs-2f718/issues/19#issuecomment-5414410967_
            @jules
